quickstack/manifests/sahara.pp

```diff
@@ -209,6 +209,13 @@
     after   => '        curr_extra = job_execution.extra.copy()',
   }
 
+  file_line { 'mr_job':
+    notify  => Service['httpd'],
+    path    => '/usr/share/openstack-dashboard/openstack_dashboard/contrib/sahara/content/data_processing/jobs/workflows/create.py',
+    line    => "                'data-jobtype-mapreduce': (\"Choose libraries\"),",
+    after   => "                'data-jobtype-java': _\u0028\"Choose libraries\"\u0029,"
```

Comment:
There seems to be no comma(,) at the end in the code. This pattern match might fail, not sure, but please check.